src/Slider/Slider.js

```diff
@@ -542,7 +547,7 @@ class Slider extends Component {
 
     const {prepareStyles} = this.context.muiTheme;
     const styles = getStyles(this.props, this.context, this.state);
-    const sliderStyles = styles.slider;
+    const sliderStyles = Object.assign(styles.slider, this.props.sliderStyle);
```

Comment:
Could we destructure the `sliderStyle` from `this.props`? That would prevent it from beeing applied to the root element.